io-console.gemspec

```
@@ -1,5 +1,13 @@
 # -*- ruby -*-
-_VERSION = "0.7.2.dev.1"
+_VERSION = ["", "ext/io/console/"].find do |dir|
+  begin
+    break File.open(File.join(__dir__, "#{dir}console.c")) {|f|
+      f.gets("\nIO_CONSOLE_VERSION ")
+      f.gets[/\s*"(.+)"/, 1]
```

`\s*` seems useless since this string doesn't start with a space.

```suggestion
      f.gets[/"(.+)"/, 1]
```
or
```suggestion
      f.gets[/\s*=\s*"(.+)"/, 1]
```

---

Thanks. I've updated.